src/Status/Parsing.elm

```diff
@@ -0,0 +1,8 @@
+module Status.Parsing exposing (statusParser)
+
+import UrlParser exposing (Parser, s, string, (</>))
```

Comment:
Good topic to discuss later as well. 
See: https://github.com/ohanhi/elm-style-guide#module-definitions